static/src/javascripts/projects/common/modules/ui/images.js

```diff
@@ -27,7 +27,7 @@ function (
                 },
                 'ui:images:lazyLoaded': function (context) {
                     picturefill({
-                        elements: [context]
+                        elements: context ? [context] : null
```

Comment:
could we not modify `upgradePictures` above to accept the context as a parameter, and use it instead of `document.body` if it exists?